pyboy/plugins/game_wrapper_super_mario_land.py

```diff
@@ -166,6 +235,13 @@ def game_area(self):
         """
         return PyBoyGameWrapper.game_area(self)
 
+    def game_over(self):
+        if self.lives_left > 0:
+            return False
+        tiles = self._game_area_np()
+        mario_at_bottom = np.any(np_in_mario_tiles(tiles[-1, :]))
+        return mario_at_bottom or np.any(tiles == 15)
```

Comment:
We might need a test to verify game_over

Reply:
Yeah but for that we need ROM manipulations of some sort, otherwise it will take a lot of time to reach the states to test (and it's hard to hard code).
Can we load a saved state for tests ?

Reply:
I guess it's ok only to test that code is working. We don't need to exhaustively test that every classification is correct.

Reply:
Maybe we could reduce his "lives"? https://datacrystal.romhacking.net/wiki/Super_Mario_Land:RAM_map